frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-secrets.feature

```diff
@@ -9,14 +9,17 @@ Feature: Secrets
 
         @smoke
         Scenario: Add Secrets: P-08-TC01
```

Comment:
```suggestion
        Scenario Outline: Add Secrets: P-08-TC01
```

Scenario Outline is required when using a examples table, right?

Reply:
Great catch, thanks

Reply:
Updated